pkg/odo/genericclioptions/context.go

```diff
@@ -56,7 +56,7 @@ func client(command *cobra.Command, shouldSkipConnectionCheck ...bool) *occlient
 		log.Errorf("client function only accepts one optional argument, was given: %v", shouldSkipConnectionCheck)
 		os.Exit(1)
 	}
-
+	skipConnectionCheck = true
```

Comment:
Isn't this always forcing connection check to be skipped even if we pass that $SkipConnectionCheckFlagName ?
Can we instead default the value of that flag to true if not the case?

Reply:
@anmolbabu , this is temporary fix, as per @kadel , we need to skip connection check for all commands

Reply:
But why not [here](https://github.com/openshift/odo/blob/e5287a04136da1c0e9cc6c31a5afe14852f9f0f5/pkg/odo/cli/cli.go#L89)

Reply:
@anmolbabu ,but then if somone accidently provides the flag, that means there will be connection check instead of skipping

Reply:
or maybe make sense also, I'll check

Reply:
it's better that we comment out that flag

Reply:
+1